modules/dynamic-plugins/ref-rh-supported-plugins.adoc

```diff
@@ -102,7 +116,7 @@
 |`./dynamic-plugins/dist/janus-idp-backstage-plugin-tekton`
 
 
-|Topology  |`https://npmjs.com/package/@janus-idp/backstage-plugin-topology/v/1.27.0[@janus-idp/backstage-plugin-topology]` |1.27.0 
+|Topology  |`https://npmjs.com/package/@janus-idp/backstage-plugin-topology/v/1.27.3[@janus-idp/backstage-plugin-topology]` |1.27.3 
```

Comment:
Same comment: worthwhile to make this an attribute?

Reply:
Same response as above ^^^